README.md

```diff
@@ -21,13 +21,54 @@ for us to support them.
 ## Installation
 
 ```bash
-asdf plugin-add php https://github.com/asdf-community/asdf-php.git
+asdf plugin-add php
 ```
 
+Since this plugin compiles PHP from source, you'll want to ensure all other required packages are installed.
+After you've installed additional system packages, install a specific PHP version:
+
+```bash
+asdf install php 7.4.16
+```
+
+You can test to ensure the installation was successful:
+
+```bash
+asdf reshim
+php --version
+```
+
+Composer is installed globally alongside PHP by default. If your application requires additional php extensions, you may need to install them via `pecl`. For example:
+
+```bash
+pecl install redis
+pecl install imagick
+
+echo "extension=redis.so
+extension=imagick.so" > $(asdf where php)/conf.d/php.ini
+```
+
+#### MacOS
+
+To install PHP on MacOS, you'll need a set of packages installed via homebrew:
+
+```bash
+brew install bison bzip2 freetype gettext libiconv icu4c jpeg libedit libpng libxml2 libzip openssl readline webp zlib
+```
```

Comment:
See here: https://github.com/asdf-community/asdf-php#prerequirements

Reply:
I did see that, but it feels a bit more user friendly to include the exact command needed for the installation here so the user doesn't have to go digging in a GH workflow config. Happy to drop this if it feels too verbose.

Reply:
I agree with that, but if it becomes obsolete and different from CI, more users will be confused. 
In many cases, we can only guarantee that a weekly scheduled build does not break our plugin. Addition, I often miss that...

Reply:
Makes sense. I removed the explicit command and just linked to the workflow area. The link contains the master sha, but I think that's an ok tradeoff: users can just switch to the master branch to get the latest brew command.